db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_11_03_140639) do
+ActiveRecord::Schema.define(version: 2019_10_30_132203) do
```

Comment:
Why is the schema version older after the change?

Reply:
That is a great question but I was told never to touch the schema file...

Reply:
I tried to do a rails db:environment:set db:drop db:create db:schema:load RAILS_ENV=test to update the schema date but it has not changed and I saw on the internet that this is a common issue with schema.

Reply:
it's allright, are your tests running green?

Reply:
yes sir :)